conference.js

```diff
@@ -1283,6 +1283,27 @@ export default {
             // FIXME close
         });
 
+        room.on(ConferenceEvents.SUSPEND_DETECTED, () => {
+            // after wake up, we will be in a state where conference is left
+            // there will be dialog shown to user, and new page need to be
+            // navigated, so we do not care about device change, which happens
```

Comment:
Maybe this comment is a little bit outdated because we are only showing an overlay and we are not navigating to different page.